docs/installation.rst

```diff
@@ -36,31 +112,27 @@ Setup AvaFrame
 
 Create a new `conda environment
 <https://conda.io/projects/conda/en/latest/user-guide/concepts/environments.html>`_
-for AvaFrame and activate it::
+for AvaFrame, activate it and install pip in this environment::
 
   conda create --name avaframe_env
   conda activate avaframe_env
+  conda install pip numpy cython
 
 Clone the AvaFrame repository (in a directory of your choice: [YOURDIR]) and change into it::
 
   cd [YOURDIR]
   git clone https://github.com/avaframe/AvaFrame.git
   cd AvaFrame
 
-AvaFrame needs following requirements as specified in the requirements.txt:
-
-.. include:: ../requirements.txt
-   :literal:
+Compile the cython com1DFA part. You might also have to install a c-compiler (gcc or similar) through your systems
+package manager::
 
-Install these with (conda-forge is needed for cmcrameri)::
+  python setup.py build_ext --inplace
 
-  conda install -c conda-forge --file requirements.txt
+.. Warning::
+   You will have to do this compilation every time something changes in the cython code.
```

Comment:
suggestion: add a hint that it is a good idea to do this every time changes are pulled from the master